src/external/lua/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.6)
```

Comment:
I use LTS to develop in, so if we merged this as is I'd not be able to compile master. Once 16.04 is released I'll upgrade so that's why I was suggesting we wait until that happens.